lib/config/presets/internal/monorepo.ts

```diff
@@ -45,6 +45,7 @@ const repoGroups = {
   'System.IO.Abstractions':
     'https://github.com/System-IO-Abstractions/System.IO.Abstractions/',
   'telus-tds': 'https://github.com/telusdigital/tds',
+  'Theme UI': 'https://github.com/system-ui/theme-ui',
```

Comment:
```suggestion
  'theme-ui': 'https://github.com/system-ui/theme-ui',
```
@rarkins i think we agreed to only use lowercase without spaces, so i suggest the original version here

Reply:
I'm actually not sure about that. First of all, spaces in the monorepo *name* will be converted to hyphen in the branch name, so it's the same result there. What is the result of this other than that the name of the monorepo in the PR title looks less recognizable?

Reply:
This is also the preset name: `monorepo:Theme UI`.

So i would now prefer to use the lowercase no space variant.